share/completions/git.fish

```diff
@@ -798,6 +798,10 @@ format:\tSpecify which information to show"
     end
 end
 
+function __fish_git_is_rebasing
+    count (string join "/" (__fish_git rev-parse --show-toplevel --git-path rebase-))* > /dev/null
```

Comment:
Very nice changes overall, thanks!

this commit can probably be dropped.
.git/rebase-merge exists during interactive rebase, while
.git/rebase-apply exists during non-interactive rebase. But --continue/--abort etc are not valid there.